src/testing.h

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (C) 2016 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#ifndef TESTING_H
+#define TESTING_H
+
+#include "clock.h"
+
+/**
+ * @brief The signature of a test function
+ * @details A graphics test is a function that represents a test of one graphics feature.
+ * It should perform draw calls to the screen that attempt to cover the different use
+ * cases of that feature.
+ * @return A value indicating whether the test was successful or failed
+ */
+typedef int (*tester)(void);
```

Comment:
Naming: I'm not sure if "tester" is enough to show that this is in fact a function. Let's name it something like `test_func_t`?